pom.xml

```diff
@@ -369,7 +372,7 @@
                             <generateApiDocumentation>false</generateApiDocumentation>
                             <generateModels>true</generateModels>
                             <modelsToGenerate>
-                              JobStatus,ExportTypeSpecificParameters,EHoldingsExportConfig,IdentifierType,EntityType,Progress,ExportType,Job,Configuration,ConfigurationCollection,LoggedObjectType,ActionType,LogRecord,logRecord_items,logRecord_linkToIds,LogRecordCollection
```

Comment:
Why is this removed?

Reply:
We have duplication 'EHoldingsExportConfig' in bursar-export-worker.
All ExportConfig schemas generates in bursar. We should split them in future